controllers/activemqartemis_reconciler.go

```diff
@@ -1630,6 +1634,23 @@ func (reconciler *ActiveMQArtemisReconcilerImpl) NewPodTemplateSpecForCR(customR
 	configMapsToCreate := []string{brokerPropertiesConfigMapName}
 	configMapsToCreate = append(configMapsToCreate, customResource.Spec.DeploymentPlan.ExtraMounts.ConfigMaps...)
 
+	loggingConfigMapName := customResource.Name + "-logging-config"
+	customLogging := false
+
+	_, err := configmaps.RetrieveConfigMap(namespacedName.Namespace, loggingConfigMapName, client)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
```

Comment:
requiring the entry"xx-logging-config" in extra mounts will give us this validation via extra mounts validation.